website/docs/cdktf/release/upgrade-guide-v0-13.mdx

```diff
@@ -0,0 +1,261 @@
+---
+page_title: Upgrading to CDKTF Version 0.13
+description: >-
+  Introducing namespaces for generated providers
+---
+
+# Upgrading to CDK for Terraform Version 0.13
+
+0.13 includes performance improvements to generated providers. Instead of exporting a flat list of exports
+including all supported resources and data sources, we now logically group contructs in namepsaces.
```

Comment:
> we now logically group contructs in namepsaces

I'm wondering whether "logically" sounds better than it actually is 😁
btw: I think we should call out somewhere that each resource has its own namespace? Or is that in the blog post / other docs?

Reply:
Hmm, that's a good point. I'll revise this a bit.